src/main/java/dev/meyi/bn/utilities/Utils.java

```diff
@@ -208,27 +221,29 @@ public static void drawCenteredString(String text, int x, int y, int color, floa
     Minecraft.getMinecraft().fontRendererObj.drawString(text, x, y, color);
     GL11.glScalef((float) Math.pow(moduleScale, -1), (float) Math.pow(moduleScale, -1), 1);
   }
-
-  public static void initializeConfigValues() {
-    if (BazaarNotifier.config.has("craftingListLength")) {
-
-      Configuration.craftingListLength = BazaarNotifier.config.getInt("craftingListLength");
-      Configuration.suggestionListLength = BazaarNotifier.config.getInt("suggestionListLength");
-      Configuration.craftingSortingOption = BazaarNotifier.config.getInt("craftingSortingOption");
-      Configuration.showInstantSellProfit = BazaarNotifier.config
-          .getBoolean("showInstantSellProfit");
-      Configuration.showSellOfferProfit = BazaarNotifier.config.getBoolean("showSellOfferProfit");
-      Configuration.showProfitPerMil = BazaarNotifier.config.getBoolean("showProfitPerMil");
-      Configuration.collectionCheckDisabled = BazaarNotifier.config
-          .getBoolean("collectionChecking");
-    } else {
-      Configuration.craftingListLength = Defaults.CRAFTING_LIST_LENGTH;
-      Configuration.suggestionListLength = Defaults.SUGGESTION_LIST_LENGTH;
-      Configuration.craftingSortingOption = Defaults.CRAFTING_SORTING_OPTION;
-      Configuration.showInstantSellProfit = Defaults.INSTANT_SELL_PROFIT;
-      Configuration.showSellOfferProfit = Defaults.SELL_OFFER_PROFIT;
-      Configuration.showProfitPerMil = Defaults.PROFIT_PER_MIL;
-      Configuration.collectionCheckDisabled = Defaults.COLLECTION_CHECKING;
+  public static void updateResources() throws IOException{
+    String result;
+    HttpGet request;
+    HttpResponse response;
+    HttpClient client = HttpClientBuilder.create().build();
+    request = new HttpGet(
+            "https://raw.githubusercontent.com/Detlev1/BazaarNotifier/master/src/main/resources/resources.json");
```

Comment:
We need to change this to the proper repository before merging & then the branch will probably also change to one that's specifically for the resources file updates (so we don't have to push to master without version changes)